otwcode/otwarchive
# AO3-5371 Add descriptive classes to external work and series blurbs

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5371

## Purpose

Add descriptive classes to series and external work blurbs. This will be reflected across different pages, including for bookmarks.

I also tried to do some internationalization in the lines I changed, and added the number delimiter to `link_to_bookmarkable_bookmarks` in `bookmarks_helper.rb`. I hope it's not out of scope?

Some points of uncertainty: 
1. I added an `inspired` class for the related works statistic — not sure about this and can revert this back if necessary.
 2. If a descriptive class is needed for the `complete` status  in series, please let me know.

## Testing Instructions

Try to create a site skin that changes the color of soecfic statistics fields in series and external work blurbs. For example:

```
.stats dt.bookmarks a,
.stats dt.words,
.stats dt.works,
.stats dt.inspired,
.stats dd.bookmarks a,
.stats dd.words,
.stats dd.works,
.stats dd.inspired {
  font-weight: bold;
  color: red;
}
```

 Make sure it works as intended.

## Credit
weeklies (she/her)
